### DISCUSSION TITLE
Screen in portrait slider track still in landscape

### DISCUSSION THREAD

**Discussion by Jwf68** [RudolphRiedel/FT800-FT813#141] [Q&A]
Hi Rudolph,
Working slider in landscape. 
Activates portrait: #define EVE_ROTATE				0x02 // screen in portrait
Slider tracker is still in portrait mode, the red area is the slider tracker. see picture.
![imageedit_2_5588151952](https://github.com/user-attachments/assets/6cd27be9-3176-49d7-bda9-9833edae2c14)

Something seems wrong.

Best regards
Jonas Wahlfrid

    EVE_cmd_dl_burst(DL_TAG); // Enable tagging    
    EVE_cmd_dl_burst(DL_TAG + SETPOINT_SLIDER); // Tag following items
    EVE_cmd_slider(300, 250, 220, 25, 0, sliderPos, 1000);
    EVE_cmd_track(300, 250, 220, 25, SETPOINT_SLIDER);
    EVE_cmd_dl_burst(DL_CLEAR_TAG); //EVE_TAG_MASK(0); // Stop following items being tagged

**Comment by RudolphRiedel** [comment]
Hi,

what is the configuration for the tracker?

**Comment by Jwf68** [comment]
The code that works in landscape but not in portrait:
EVE_cmd_dl_burst(DL_TAG); // Enable tagging    
EVE_cmd_dl_burst(DL_TAG + SETPOINT_SLIDER); // Tag following items
EVE_cmd_slider(300, 250, 220, 25, 0, sliderPos, 1000);
EVE_cmd_track(300, 250, 220, 25, SETPOINT_SLIDER);
EVE_cmd_dl_burst(DL_CLEAR_TAG); //EVE_TAG_MASK(0); // Stop following items being tagged

**Comment by RudolphRiedel** [comment]
Oh yes, if you are using the macro, are you sending the touch calibration values for the rotated screen?

**Comment by Jwf68** [comment]
No I am using the touch calibration values for landscape.

**Comment by RudolphRiedel** [comment]
That is the issue, the rotation also rotates the calibration, you either need to rotate after calibration or use a different set of calibration values.

Try this:
        touch_calibrate();
        EVE_cmd_setrotate(2);

**Comment by Jwf68** [comment]
Perfekt I will try that tomorrow. Thank you.

**Comment by Jwf68** [comment]
A different set of calibration values works fine :-)

**Comment by Jwf68** [comment]
tff.c
```
#if defined (EVE_ME812A)
    #if EVE_ROTATE == 0x02
        // portrait    
        EVE_memWrite32(REG_TOUCH_TRANSFORM_A, 0xfffffd4c);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_B, 0x0000897f);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_C, 0xffe6c9f4);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_D, 0x0000db8a);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_E, 0x0000044c);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_F, 0xffd82d36);
    #else
        // landscape     
        EVE_memWrite32(REG_TOUCH_TRANSFORM_A, 0x0000d7f9);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_B, 0xffffff09);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_C, 0xffe66cf0);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_D, 0x0000026e);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_E, 0xffff7c37);
        EVE_memWrite32(REG_TOUCH_TRANSFORM_F, 0x01eb8d19);
    #endif
#endif
```
